google-map-marker.html

```diff
@@ -249,6 +249,14 @@
           },
 
           /**
+           * The marker's label.
+           */
+          label: {
+            type: String,
+            value: null,
+            observer: '_labelChanged'
+          },
```

Comment:
Oops! It seems that there are spaces instead of clean break line 😅

![feb-01-2017 16-58-23](https://cloud.githubusercontent.com/assets/1007051/22514469/b4cfdbe8-e89f-11e6-9292-65d5a28d13c0.gif)